frontend/src/components/form/api/ApiWrapper.vue

```diff
@@ -159,11 +159,12 @@ export default {
     },
     // reload data from API (doesn't force loading from server if available locally)
     reload () {
-      this.isLoading = true
       this.resetErrors()
+      const obj = this.api.get(this.uri)
+      this.isLoading = obj._meta.loading
```

Comment:
Wäre es nicht besser, `isLoading` als `computed` zu berechnen? Dann musst du nicht imperativ überall dran denken, diesen Wert mit dem Store synchron zu halten.
```
  isLoading() {
    return this.api.get(this.uri)._meta.loading
  }
```
Das lädt zwar direkt nach dem mounten spätestens zu ersten mal die URI, aber ich denke das wollen wir sowieso, und hal-json-vuex ist schlau genug, für dieselbe URI nur 1 Request aufs Mal abzusetzen.

Reply:
Da bin ich sehr unsicher - da die Methode `reload()` nicht zwingend aufgerufen wird (siehe `created()`).
Ein `computed` würde aber immer ein Store-Zugriff verursachen.

Das Verhalte wäre somit nicht das gleiche.

Reply:
Bin nicht sicher ob du das meinst, aber ein `computed` wird erst evaluiert, wenn es das erste Mal gebraucht wird: https://github.com/vuejs/vue/issues/7672
Aber das `isLoading` Flag wird ja schon von Anfang an (im Template) verwendet und hat den Initialwert `false`, was im allgemeinen Fall oft nicht korrekt ist. Dieses Flag soll ja in dieser Komponente wirklich signalisieren, ob die darunterliegenden Daten noch nicht fertig geladen sind, und das deckt sich 1:1 mit dem `loading` Flag welches bereits jederzeit im Store abrufbar ist.

Ich kann meinen Vorschlag aber auch gut mal isoliert in einen PR verpacken, kein Grund diesen PR nicht zu mergen.